lib/rsgem/colors.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RSGem
+  class Colors
+    class << self
+      def colorize(string, color)
+        "\e[#{color_code(color)}m#{string}\e[0m"
+      end
+
+      def color_code(color)
+        {
+          red: 31,
+          green: 32,
+          yellow: 33,
+          blue: 34,
+          white: 97,
+        }[color] || 39
```

Comment:
Maybe we can make this hash a constant in this class to avoid allocating memory for it each time we colorize something (I know it shouldn't matter the memory consumption, so maybe from a code style perspective a think a constant would be more suitable in here)

```ruby
module RSGem
  class Colors
    COLORS = {
      red: 31,
      # ...
    }.freeze

    ...

    def color_code(color)
      COLORS[color] || 39
    end
```

Something like this ^^^ maybe (?)

Reply:
Yeah I like that better, I'll change it to a constant. Any other colors you think we should add while I'm here?